packages/app/src/sections/dashboard/RewardsTab.tsx

```diff
@@ -0,0 +1,352 @@
+import { ZERO_WEI } from '@kwenta/sdk/constants'
+import { formatDollars, formatPercent, truncateNumbers } from '@kwenta/sdk/utils'
+import { wei } from '@synthetixio/wei'
+import { FC, useCallback, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import Button from 'components/Button'
+import { FlexDivCol, FlexDivRow, FlexDivRowCentered } from 'components/layout/flex'
+import LabelContainer from 'components/Nav/DropDownLabel'
+import Select, { DropdownIndicator, IndicatorSeparator } from 'components/Select'
+import { Body, Heading } from 'components/Text'
+import { EXTERNAL_LINKS } from 'constants/links'
+import { NO_VALUE } from 'constants/placeholder'
+import useIsL2 from 'hooks/useIsL2'
+import { TradingRewardProps, useEstimatedReward } from 'queries/staking/utils'
+import { StakingCard } from 'sections/dashboard/Stake/card'
+import { selectFuturesFees, selectFuturesFeesForAccount } from 'state/futures/selectors'
+import { useAppDispatch, useAppSelector } from 'state/hooks'
+import { claimMultipleAllRewards } from 'state/staking/actions'
+import { setSelectedEpoch } from 'state/staking/reducer'
+import {
+	selectEpochData,
+	selectKwentaRewards,
+	selectOpRewards,
+	selectSelectedEpoch,
+	selectSnxOpRewards,
+} from 'state/staking/selectors'
+import media from 'styles/media'
+
+type EpochValue = {
+	period: number
+	start: number
+	end: number
+	label: string
+}
+
+const RewardsTabs: FC<TradingRewardProps> = ({ period = 0 }) => {
```

Comment:
The files called RewardsTab but here it's named RewardsTabs and can see there's already a file called RewardsTabs. Also if this staking specific maybe it should be under dashboard/staking .

Reply:
Yes. Good catch. This is copied from the `RewardsTabs.tsx`. Need to clean this a bit.
![image](https://github.com/Kwenta/kwenta/assets/4819006/ca552cb3-457e-40c0-9875-582835a0961f)